tilauskasittely/lisaarivi.inc

```diff
@@ -2495,7 +2495,8 @@ if ($toimittamatta > 0 or trim($var) != '' or $rivityyppi == 'O') {
       $yhtiorow['tee_siirtolista_myyntitilaukselta'] == 'K' and
       $laskurow["tilaustyyppi"] != "U" and
       ($_tarjouksella or $_osto_myyntitilaukselta or $_extranetista) and
-      !in_array($var, array('V', 'H', 'P'))) {
+      !in_array($var, array('V', 'H', 'P'))
+      and $kpl > 0) {
```

Comment:
@tonyselkamo @hannahagberg Tonza miten tämä vaikuttaa tähän? Positiivisesti, vai onko tarkoitus, että hyvitysriveillekin jumpataan näitä automaattisia siirtojuttuja?